spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -285,8 +287,10 @@
 "LGPL, version 3.0": LGPL-2.1-only
 "LGPL/MIT": LGPL-3.0-only OR MIT
 "LGPLv3 or later": LGPL-3.0-or-later
+"LPGL, see LICENSE file.": LGPL-3.0-only
 "Lesser General Public License (LGPL)": LGPL-2.1-only
 "Lesser General Public License, version 3 or greater": LGPL-3.0-or-later
+"License :: OSI Approved :: MIT License (http://opensource.org/licenses/MIT)": MIT
```

Comment:
I believe we had this discussion before: Python-specific classifier prefixes should not be part of the mapping. I.e. this mapping should only be for "MIT License (http://opensource.org/licenses/MIT)".

Reply:
@sschuberth  This mapping is needed as authors actually use the classifiers value within the `license` field - I know this is stupid but ORT currently does remove `License :: OSI Approved :: ` from the `license` field value.

Reply:
Strictly speaking ORT does not remove the full `License :: OSI Approved :: `, it only removes a `License :: ` prefix and for those lines where the prefix was stripped it takes the last component after ` :: ` as the license (unless that last component happens to be just `OSI Approved`). But anyway, as ORT already does this cleanup, why to we need a mapping?

Reply:
We already have an entry: `MIT License (http://opensource.org/licenses/MIT)": MIT`.

Reply:
Yes, that's why I wrote "But that mapping we already have" above 😉

Reply:
Seems I haven't read your first comment (carefully).

Reply:
@sschuberth - I believe we both missed the point of Thomas' commit message.
It seems there that the string from the added mapping entry is used as `declared license`, not as `classifier`.

As according to @tsteenbe this was seen already more than once, I'd propose that we apply a similar logic as we do to the classifiers also to the `declared license`, which strips prefixes like `License :: OSI Approved :: `. Do you agree?

Reply:
My initial reaction in the link above was that such packages are simply broken and require a curation. But as it's easy to add a work-around I'm proposing https://github.com/oss-review-toolkit/ort/pull/2942. That should avoid the need for that mapping, right?

Reply:
https://github.com/oss-review-toolkit/ort/pull/2942 is merged now so this mapping should not be necessary anymore.

Reply:
Removed commit as #2942 fixes the issue.